Opc.Ua.CloudLib.Client/UACloudLibClient.cs

```diff
@@ -169,7 +170,7 @@ private async Task<T> SendAndConvertAsync<T>(GraphQLRequest request)
             }
             GraphQLResponse<JObject> response = await _client.SendQueryAsync<JObject>(request).ConfigureAwait(false);
 
-            if (response?.Errors?.Length > 0)
+            if (response.Errors?.Length > 0)
```

Comment:
why remove the ? for additional safety?

Reply:
This triggered a warning in SonarCloud on my PC. I'll put it back.

Reply:
Added explicit null check instead.